dm/config/task.go

```diff
@@ -250,34 +250,48 @@ const (
 	LoadModePhysical = "physical"
 )
 
-// DuplicateResolveType defines the duplication resolution when meet duplicate rows.
+// DuplicateResolveType defines the duplication resolution when meet duplicate rows for logical import.
 type DuplicateResolveType string
 
 const (
 	// OnDuplicateReplace represents replace the old row with new data.
 	OnDuplicateReplace DuplicateResolveType = "replace"
 	// OnDuplicateError represents return an error when meet duplicate row.
-	OnDuplicateError = "error"
+	OnDuplicateError DuplicateResolveType = "error"
 	// OnDuplicateIgnore represents ignore the new data when meet duplicate row.
-	OnDuplicateIgnore = "ignore"
+	OnDuplicateIgnore DuplicateResolveType = "ignore"
+)
+
+// PhysicalDuplicateResolveType defines the duplication resolution when meet duplicate rows for physical import.
+type PhysicalDuplicateResolveType string
+
+const (
+	// OnDuplicateNone represents do nothing when meet duplicate row and the task will continue.
+	OnDuplicateNone PhysicalDuplicateResolveType = "none"
+	// OnDuplicateManual represents that task should be paused when meet duplicate row to let user handle it manually.
+	OnDuplicateManual PhysicalDuplicateResolveType = "manual"
 )
 
 // LoaderConfig represents loader process unit's specific config.
 type LoaderConfig struct {
-	PoolSize    int                  `yaml:"pool-size" toml:"pool-size" json:"pool-size"`
-	Dir         string               `yaml:"dir" toml:"dir" json:"dir"`
-	SQLMode     string               `yaml:"-" toml:"-" json:"-"` // wrote by dump unit (DM op) or jobmaster (DM in engine)
-	ImportMode  LoadMode             `yaml:"import-mode" toml:"import-mode" json:"import-mode"`
-	OnDuplicate DuplicateResolveType `yaml:"on-duplicate" toml:"on-duplicate" json:"on-duplicate"`
+	PoolSize   int      `yaml:"pool-size" toml:"pool-size" json:"pool-size"`
+	Dir        string   `yaml:"dir" toml:"dir" json:"dir"`
```

Comment:
The `dir` seems a directory that maintains the export data. Do we need an extra config field for sst files?

Reply:
Thanks! I'll check it with PM later, maybe we need add more fields in future

Reply:
And also, lightning will check whether the source-dir and sst-dir are on the same disk (for IO performance sake, I think). I'm wondering if we should check it here

Reply:
precheck will be ported in another PR